packages/volto/src/components/manage/Contents/ContentsItem.jsx

```diff
@@ -178,6 +178,14 @@ export const ContentsItemComponent = ({
                   <FormattedMessage id="Scheduled" defaultMessage="Scheduled" />
                 </Button>
               )}
+            {item.is_working_copy && (
+              <Button className="button-margin working-copy" size="mini">
+                <FormattedMessage
+                  id="Working copy"
+                  defaultMessage="Working copy"
+                />
+              </Button>
```

Comment:
button or label: I would say this, and the two buttons preceding, are circular labels. Not buttons, as there is no action to take.
See https://react.semantic-ui.com/elements/label/#variations-circular

Reply:
I would prefer that too but I was following the pattern of the existing indicators for expired and scheduled content. We can fix it in Seven but I won't change the existing markup here.

Reply:
I understand your argument.